bazel/digest.go

```diff
@@ -45,3 +45,7 @@ func DigestToStr(d *remoteexecution.Digest) string {
 	}
 	return fmt.Sprintf("%s/%d", d.GetHash(), d.GetSizeBytes())
 }
+
+func IsEmptyDigest(d *remoteexecution.Digest) bool {
+	return d.GetHash() == EmptySha && d.GetSizeBytes() == EmptySize
```

Comment:
Is it a possibility/problem if hash is EmptySha but size is not empty?

Reply:
It's unlikely but possible that could get passed somehow, but it's essentially an invalid digest and a sign that something was not computed or passed in correctly. Checking it is on the pedantic side but won't hide any errors that should surface because of it.